internal/observability/event/eventer.go

```diff
@@ -121,15 +146,31 @@ func NewEventer(log hclog.Logger, c EventerConfig, opt ...Option) (*Eventer, err
 		b = eventlogger.NewBroker()
 	}
 
-	e := &Eventer{
-		logger: log,
-		conf:   c,
-		broker: b,
-	}
-
+	// this is simply a way to reconcile the timestamps for "now" that the
+	// broker adds to the events with a known value.
 	if !opts.withNow.IsZero() {
-		e.broker.StopTimeAt(opts.withNow)
+		b.StopTimeAt(opts.withNow)
 	}
+	return b
+}
+
+type pipeline struct {
```

Comment:
If we go forward this PR I would really like to update `pipeline` and `pipelines` to be a little bit more.. differently named?